pkg/syndesis/action/install.go

```diff
@@ -56,7 +90,7 @@ func (a *Install) Execute(syndesis *v1alpha1.Syndesis) error {
 	}
 
 	// Installation completed, set the next state
-	target := syndesis.DeepCopy()
+	target := originalSyndesis.DeepCopy()
```

Comment:
don't you miss the routeHostName when you clone from the `originalSyndesis` ? Actually I wonder why we can't just reuse syndesis and set the status on this already cloned resource.

Reply:
Actually it was on purpose.. I wanted to always leave the configuration in `spec` as it had been set by the user.

Reply:
I think this is dark area. I often see also `spec:` fields in K8s which get updated and then are e.g. also marked as `read-only`. Not sure what we should do here and what the rationale is.

* If the resource should be read and then applied to a different cluster it would make sense to leave out the route.
* If we want an easy to see where the entry point for the app is (without going hunting for the proper route), it would make sense to add the route *somewhere*.

Wdyt about adding the route (with protocol) as an annotation `io.syndesis/url` ? Or in the status, but I think its better suited for the metadata, or ?

If so, I would pick that up, so that we merge the PR now. 'want to code a little bit, too :)